test/common.jl

```diff
@@ -1,16 +1,24 @@
-using TaylorIntegration, Test, DiffEqBase
+using Test, DiffEqBase, TaylorIntegration
```

Comment:
Does the order matter here? If so, i think it is worth adding  a couple of sentences to the docs.

Reply:
The order makes no difference, I just forgot to leave that line untouched. I've now gone back to the original order, so that this line doesn't show up when doing diffs in the future